src/System.Private.CoreLib/src/System/Reflection/Emit/ConstructorBuilder.cs

```diff
@@ -190,11 +190,13 @@ public Module GetModule()
             return m_methodBuilder.GetModule();
         }
 
+#pragma warning disable CS8609 // TODO-NULLABLE: Covariant return types (https://github.com/dotnet/roslyn/issues/23268)
```

Comment:
This should not be needed now that `MethodBuilder.ReturnType` was changed to be non-nullable.

Also the "This always returns null. Is that what we want?" comment should be deleted.

Reply:
Fixed.